rbd.c

```diff
@@ -983,6 +1043,14 @@ static int tcmu_rbd_open(struct tcmu_device *dev, bool reopen)
 				    tcmu_dev_get_block_size(dev), false);
 	tcmu_dev_set_write_cache_enabled(dev, 0);
 
+	tcmu_rbd_rm_stale_entries_from_blacklist(dev);
+
+	/* Get current entry address for the image */
+	ret = rados_getaddrs(state->cluster, &state->addrs);
```

Comment:
Nit: need to condition this usage on this API being available.

Reply:
I have raise one PR for this : https://github.com/ceph/ceph/pull/36488